chrome/content/zotero/zoteroPane.js

```diff
@@ -158,6 +158,17 @@ var ZoteroPane = new function()
 		
 		Zotero.Keys.windowInit(document);
 		
+		let sortSubmenuKeys = document.getElementById('sortSubmenuKeys');
+		for (let i = 0; i < 10; i++) {
+			let key = document.createElement('key');
+			key.id = 'key_sortCol' + i;
+			key.setAttribute('modifiers', Zotero.isMac ? 'accel alt control' : 'accel alt');
```

Comment:
Windows Explorer doesn't have sort keyboard shortcuts to copy here, but Ctrl-Alt-[num] isn't used system-wide and seems common for numbered presets and things like that.